src/main/java/hudson/plugins/plot/Plot.java

```diff
@@ -399,7 +399,7 @@ public Double getDoubleFromString(String input) {
             try {
                 result = Double.parseDouble(input);
             } catch (NumberFormatException nfe) {
-                LOGGER.log(Level.INFO, "Failed to parse Double value from String."
+                LOGGER.log(Level.FINE, "Failed to parse Double value from String."
```

Comment:
@gjabouley-invn hey. have you tried actually adjusting the logic and properly verifying the `input` for emptiness? That was suggest in the ticket as well and looks legit.

Reply:
I've made corresponding change, it should now be aligned with your expectations